ethereum/Dockerfile

```diff
@@ -26,7 +27,7 @@ RUN if [ -e /certs/cert.pem ]; then git config --global http.sslCAInfo /certs/ce
 # Only invalidate the npm install step if package.json changed
 COPY --chown=node:node package.json .
 COPY --chown=node:node package-lock.json .
-COPY --chown=node:node .env.test .env
+COPY --from=const-export --chown=node:node .env.0x .env
```

Comment:
I think this may not have the arguments from the top of the `.env.test` file. Perhaps there is a better way to handle this, loading a `.env.dev` with everything but the chain registrations, and then `.env.0x` with only the chain registrations.

Reply:
It actually does since we copy `.env.test` and then update/append the just registrations (leaving the constants at the top untouched) https://github.com/wormhole-foundation/wormhole/blob/455fdfef7ef6de8ae855fcceffd7c0f7569ed92f/scripts/guardian-set-init.sh#L16.

Also tested this by running `scripts/guardian-set-init.sh` locally to make sure